README.md

```diff
@@ -8,9 +8,8 @@ While several approaches for integrating symbolic and sub-symbolic models have b
 ## Contents
 
 - [Getting Started](https://github.com/HLR/DomiKnowS/blob/Doc/User%20API/Getting%20Started.md): Provides detailed instructions on how to get started with DomiKnowS, including installation, setting up the environment, and basic usage.
-- [Documentation](https://github.com/HLR/DomiKnowS/tree/Doc/User%20API): Provides comprehensive documentation on the DomiKnowS, including classes, methods, and their usage.
+- [Documentation](https://github.com/HLR/DomiKnowS/tree/Doc/): Provides comprehensive documentation on the DomiKnowS, including classes, methods, and their usage.
```

Comment:
This is still going to the Doc branch and containing the same issues we discussed before, this should go to the class definitions I guess directly or the component documentation, Is the URL correct now?!